lib/index.js

```diff
@@ -107,7 +107,7 @@ function renderFile (root, obj) {
 }
 
 function renderUrl (obj) {
-  return rest(obj.url).then((res) => { return JSON.parse(res.entity) })
+  return rest(obj.url).then((res) => { return JSON.parse(transformRaw(obj, res.entity)) })
```

Comment:
Ok so in this case the `transformRaw` option would only work for url sources. I think this is ok, but should be noted in the docs!